cabal-install/Distribution/Client/CmdInstall/EnvironmentParser.hs

```diff
@@ -0,0 +1,72 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RecordWildCards #-}
+module Distribution.Client.CmdInstall.EnvironmentParser 
```

Comment:
Shouldn't this be in Distribution.Simple.GHC?

Reply:
I wasn't sure if it was of general enough applicability, but it's easy enough to move it.